specification/containerregistry/resource-manager/readme.typescript.md

```diff
@@ -0,0 +1,15 @@
+## TypeScript
+
+These settings apply only when `--typescript` is specified on the command line.
+Please also specify `--typescript-sdks-folder=<path to root folder of your azure-sdk-for-js clone>`.
+
+``` yaml $(typescript)
+input-file:
+  - Microsoft.ContainerRegistry/stable/2017-10-01/containerregistry.json
+  - Microsoft.ContainerRegistry/stable/2018-09-01/containerregistry_build.json
```

Comment:
Please investigate why these aren't in the readme.md file. This is suspicious.

Reply:
It was added in Node spec in this commit https://github.com/Azure/azure-rest-api-specs/commit/d5e4013d7ec88bca201ab4584fd81be835b2bfa4 but it's hard for me to find the reasoning